### PR TITLE
docs: added rust docs

### PR DESCRIPTION
**Pull request by NewtTheWolf**
@geclos idk about the Slack link, just comment on it

i think everything should be documented in here

### CODE REVIEW

**Comment by NewtTheWolf** [comment]
maybe i should add a disclaimer that this is community driven & link to the crate page and github repo @geclos ?

File: docs/guides/sdk/rust-sdk.mdx [hunk 1]
```diff
@@ -0,0 +1,286 @@
+---
+title: Rust SDK
+description: Learn how to use the SDKs to interact with the Latitude API.
+---
+
+## Latitude SDK Documentation
+
+The Latitude SDK provides an easy way to interact with the Latitude API, allowing you to run documents and chat with AI models.
+
+You can reach out to us with any question or request through our [Slack channel](https://join.slack.com/t/trylatitude/shared_invite/zt-17dyj4elt-rwM~h2OorAA3NtgmibhnLA).
```
Comment (geclos) [inline docs/guides/sdk/rust-sdk.mdx:10]: no need for this yeah

**Comment by geclos** [comment]
> maybe i should add a disclaimer that this is community driven & link to the crate page and github repo @geclos ?

yes i think this is a great idea let's do it

File: docs/guides/sdk/rust-sdk.mdx (hunk 1 above)
Comment (NewtTheWolf) [reply docs/guides/sdk/rust-sdk.mdx]: updated

**Comment by geclos** [comment]
thanks!

**Pull request merged**